lib/network_utils.ml

```diff
@@ -821,6 +821,15 @@ module Sysctl = struct
   let write value variable =
     ignore (call_script sysctl ["-q"; "-w"; variable ^ "=" ^ value])
 
+  let set_ipv6_enabled interface value =
```

Comment:
Minor suggestions:
* use `enabled` instead of `value`
* simplify `when value = true` to `when enabled`
* use `printf` to build the string for `variable`
* is `if value then "0" else "1"` correct? It writes `0` for `false`

Reply:
I agree with the above, except for the last point.

Reply:
Sorry, it writes `0` for `true`, which is counter-intuitive.

Reply:
Yes, the sysctl is "disable_ipv6", and it is (on XenServer) set to 1 by default.  To enable IPv6, you set disable_ipv6=0.  I could have named the function "set_ipv6_disabled" and had the sense match that of the sysctl, but I thought that was more confusing!

I'll try to make the code a bit clearer.